qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -171,8 +171,8 @@ def run_calibration(ceria_ws, calibration, full_instrument_cal_ws):
         cal_table, diag_ws, mask = PDCalibration(InputWorkspace=foc_name, OutputCalibrationTable=cal_table_name,
                                                  DiagnosticWorkspaces=diag_ws_name,
                                                  PeakPositions=EnggUtils.default_ceria_expected_peaks(final=True),
-                                                 TofBinning=[15500, -0.0003, 52000],
-                                                 PeakWindow=0.04,
+                                                 TofBinning=[12000, -0.0003, 52000],
```

Comment:
What's the reason for this change to the rebin params?

Reply:
The binning occurs in TOF, the original were good for a whole bank but not for some of the texture groups (would chop off the peak at 0.85 Ang in half the spectra). 

![image](https://user-images.githubusercontent.com/55979119/144600842-f7b0ae14-e5a6-468e-96ec-4057edec2aca.png)

I then added another peak at lower d-spacing (see below) and so the limit is now considerably lower.